non/spire
# Microsite

How to test locally:
- `sbt docs/makeMicrosite`
- `cd docs/target/site; jekyll serve`
- open http://127.0.0.1:4000/spire/

See #702
